server/sonar-server/src/main/java/org/sonar/server/projectbranch/ws/ListAction.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonar.server.projectbranch.ws;
 
+import com.google.common.base.Optional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.base.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)